.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: node_js
+node_js:
+  - "10"
+  - "12"
+  - "13"
```

Comment:
I think tests should just be even numbered releases and the latest via "node" (which is "13" right now).  "13" will become unsupported and useless before "10" so less to update later if we just leave it out now.